client/thin-replica-client/src/trc_hash.cpp

```diff
@@ -52,19 +52,19 @@ static string hashUpdateFromEntryHashes(uint64_t block_id, const map<string, str
   string concatenated_entry_hashes;
   concatenated_entry_hashes.reserve(sizeof(block_id) + 2 * entry_hashes.size() * kThinReplicaHashLength);
 
-#ifdef BOOST_LITTLE_ENDIAN
+#ifdef BOOST_ENDIAN_LITTLE_BYTE
   concatenated_entry_hashes.append(reinterpret_cast<const char*>(&block_id), sizeof(block_id));
-#else  // BOOST_LITTLE_ENDIAN not defined in this case
-#ifndef BOOST_BIG_ENDIAN
+#else  // BOOST_ENDIAN_LITTLE_BYTE not defined in this case
+#ifndef BOOST_ENDIAN_BIG_BYTE
```

Comment:
What is the meaning of the new names?
Was the rename necessary?

Reply:
definition move from `<boost/detail/endian.hpp>` to `<boost/predef/other/endian.h>`
I was sure I redefined them in our  `util/endianess.hpp` header, but it disappeared somehow.
Will fix.